install.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+
+# CHECKS FOR SUDO ACCESS (added because of those who have seperate partitions for /home and /root) 
+
+if [ "$EUID" -eq 0 ]
+  then echo "Please don't run as root" && exit
+  exit
+fi
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+cd $(mktemp -d)
+
+# Install git and wget
+
+echo "What distro do you use?"
+echo "1) Arch Linux"
+echo "2) Fedora Linux"
+echo "3) Gentoo/Funtoo Linux"
+echo "4) Ubuntu/Debian and Ubuntu/Debian based distros"
+echo "5) Void Linux"
+echo "6) NixOS"
+echo "7) Alpine Linux"
+echo "8) openSUSE Linux"
+read -p "Please input a number between 1 and 8: " DISTRO
+case $DISTRO in 
+[1])
+	sudo pacman -S git wget unzip --needed
+	;;
+
+[2])
+  sudo dnf install wget git unzip
+	;;
+
+[3])
+  sudo emerge wget git unzip
+  ;;
+
+[4])
+	sudo apt install wget git unzip
+	;;
+  
+[5])
+  sudo xbps-install wget git unzip
+  ;;
+  
+[6])
+  sudo nix-shell -p wget git unzip
+  ;;
+  
+[7])
+  sudo apk add wget git unzip
+  ;;
+  
+[8])
+  sudo zypper install wget git unzip
+  ;;
+esac
+
+# Download, unzip, and execute SFP (formerly Steam Friends Patcher)
+
+wget https://github.com/PhantomGamers/SFP/releases/download/0.0.14/SFP_UI-SelfContained-linux-x64.zip
+unzip SFP_UI-SelfContained-linux-x64.zip
+chmod +x SFP_UI
+./SFP_UI
+
+# Download the original Steam for Metro (Will create a directory for it so that things stay clean) 
+
+wget https://github.com/minischetti/metro-for-steam/archive/refs/tags/v4.4.zip
+mkdir ./metro && mv v4.4.zip ./metro/ && cd ./metro
+
+# Unzip Metro and remove the ZIP file
+
+unzip ./v4.4.zip
+rm -f ./v4.4.zip
+
+# Ask for the path to the users .steam file so that it can be used with Flatpak installations,
+# then makes a folder in the .steam/steam/skins directory called 'metro' to unzip metro to.
+# Also makes the skins folder if it doesn't exist
+
+read -p "Where is your steam installation? Copy the path (e.g. /home/thebearodactyluwu/.steam) and paste it here: " steaminstall
```

Comment:
Couple things:
1. This could be mostly automated by checking for the .steam folder in $HOME, and only running this if not found
2. The .steam folder isn't the location of the installation, it just has a bunch of symlinks. Most users (including myself) will supply .steam/steam instead of just .steam-- there should be a check in place to handle that.

Reply:
I'm adding that right now